StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerFooterView.swift

```diff
@@ -69,7 +69,16 @@ final class LinkAccountPickerFooterView: UIView {
         didSelectConnectAccount()
     }
 
-    func enableButton(_ enableButton: Bool) {
-        connectAccountButton.isEnabled = enableButton
+    func didSelectedAccount(_ selectedAccountTuple: FinancialConnectionsAccountTuple?) {
+        if
+            let selectedAccountTuple = selectedAccountTuple,
+            let selectionCta = selectedAccountTuple.accountPickerAccount.selectionCta
```

Comment:
can this two line be replaced with `let selectionCta = selectedAccountTuple?.accountPickerAccount.selectionCta`

Reply:
Sounds good!